doc/build_doc/linux/README.md

```diff
@@ -291,115 +291,26 @@ cd openvino-models
-│   │   ├── [ 16M]  lm_heads.bin
-│   │   ├── [ 11K]  lm_heads.xml
-│   │   ├── [672M]  musicgen_decoder_combined_weights.bin
-│   │   ├── [337M]  musicgen_decoder_combined_weights_int8.bin
-│   │   ├── [2.5M]  musicgen_decoder_static0_10s.xml
-│   │   ├── [2.5M]  musicgen_decoder_static0_5s.xml
-│   │   ├── [3.0M]  musicgen_decoder_static_batch1_int8.xml
-│   │   ├── [2.5M]  musicgen_decoder_static_batch1.xml
-│   │   ├── [3.0M]  musicgen_decoder_static_int8.xml
-│   │   ├── [2.5M]  musicgen_decoder_static.xml
-│   │   └── [8.0M]  sinusoidal_positional_embedding_weights_2048_1024.raw
-│   ├── [775K]  musicgen-small-tokenizer.bin
-│   ├── [5.7K]  musicgen-small-tokenizer.xml
-│   ├── [4.0K]  stereo
-│   │   ├── [ 32M]  embed_tokens.bin
-│   │   ├── [ 28K]  embed_tokens.xml
-│   │   ├── [3.0M]  enc_to_dec_proj.bin
-│   │   ├── [2.7K]  enc_to_dec_proj.xml
-│   │   ├── [192M]  initial_cross_attn_kv_producer.bin
-│   │   ├── [145K]  initial_cross_attn_kv_producer.xml
-│   │   ├── [ 32M]  lm_heads.bin
-│   │   ├── [ 21K]  lm_heads.xml
-│   │   ├── [672M]  musicgen_decoder_combined_weights.bin
-│   │   ├── [337M]  musicgen_decoder_combined_weights_int8.bin
-│   │   ├── [2.5M]  musicgen_decoder_static0_10s.xml
-│   │   ├── [2.5M]  musicgen_decoder_static0_5s.xml
-│   │   ├── [3.0M]  musicgen_decoder_static_batch1_int8.xml
-│   │   ├── [2.5M]  musicgen_decoder_static_batch1.xml
-│   │   ├── [3.0M]  musicgen_decoder_static_int8.xml
-│   │   ├── [2.5M]  musicgen_decoder_static.xml
-│   │   └── [8.0M]  sinusoidal_positional_embedding_weights_2048_1024.raw
-│   ├── [209M]  t5.bin
-│   └── [550K]  t5.xml
-└── [4.0K]  openvino-models
-    ├── [4.0K]  deepfilternet2
-    │   ├── [3.2M]  df_dec.bin
-    │   ├── [112K]  df_dec.xml
-    │   ├── [2.5M]  enc.bin
-    │   ├── [175K]  enc.xml
-    │   ├── [3.2M]  erb_dec.bin
-    │   └── [181K]  erb_dec.xml
-    ├── [4.0K]  deepfilternet3
-    │   ├── [3.2M]  df_dec.bin
-    │   ├── [123K]  df_dec.xml
-    │   ├── [1.8M]  enc.bin
-    │   ├── [186K]  enc.xml
-    │   ├── [3.1M]  erb_dec.bin
-    │   └── [185K]  erb_dec.xml
-    ├── [141M]  ggml-base.bin
-    ├── [ 39M]  ggml-base-encoder-openvino.bin
-    ├── [281K]  ggml-base-encoder-openvino.xml
-    ├── [465M]  ggml-small.bin
-    ├── [168M]  ggml-small-encoder-openvino.bin
-    ├── [804K]  ggml-small-encoder-openvino.xml
-    ├── [465M]  ggml-small.en-tdrz.bin
-    ├── [168M]  ggml-small.en-tdrz-encoder-openvino.bin
-    ├── [512K]  ggml-small.en-tdrz-encoder-openvino.xml
-    ├── [ 96M]  htdemucs_v4.bin
-    ├── [1.8M]  htdemucs_v4.xml
-    ├── [8.2M]  noise-suppression-denseunet-ll-0001.bin
-    └── [674K]  noise-suppression-denseunet-ll-0001.xml
-
-
-
-7 directories, 74 files
-```
+# clone the HF repo
+git clone https://huggingface.co/Intel/versatile_audio_super_resolution_openvino
 
+# unzip the 'base' set of models into audiosr
+unzip versatile_audio_super_resolution_openvino\versatile_audio_sr_base_openvino_models.zip -d openvino-models\audiosr
```

Comment:
Slashes are backwards for Linux in this section. Should be / vs current \. Modifying this in the README will make cut-n-paste easier.

Reply:
Ah, thanks -- fixed.